app/src/main/java/manparvesh/ideatrackerplus/recycler/RecyclerOnClickListener.java

```diff
@@ -71,7 +73,8 @@ public static void setPrimaryColor(int color) {
      */
     private void showIdeaDialog() {
 
-        String text = mDbHelper.getTextById(mIdRecycler);
+        SpannableString text = new SpannableString(mDbHelper.getTextById(mIdRecycler));
+        text.setSpan(new AbsoluteSizeSpan(24,true),0,text.length(),0);
```

Comment:
Better if you can manage the spacing so code looks good,
`text.setSpan(new AbsoluteSizeSpan(24, true), 0, text.length(), 0);`

Reply:
Yeah Sure, thanks for the tip 👍  . Pushed the commit.